nvdaHelper/vbufBase/backend.cpp

```diff
@@ -249,3 +236,19 @@ VBufBackend_t::~VBufBackend_t() {
 	LOG_DEBUG(L"base Backend destructor called"); 
 	nhAssert(runningBackends.count(this) == 0);
 }
+
+VBufStorage_fieldNode_t* VBufBackend_t::reuseExistingNodeInRender(int docHandle, int ID)  {
+	auto existingNode=this->getControlFieldNodeWithIdentifier(docHandle,ID);
+	if(existingNode) {
+		auto i=std::find(this->workingInvalidSubtreesList.begin(),this->workingInvalidSubtreesList.end(),existingNode);
+		if(i==this->workingInvalidSubtreesList.end()) {
+			// This existing node was not marked as invalid, therefore it can be re-used.
+			return existingNode;
+		} else {
+			// this existing node was marked as invalid also, therefore we can re-render it here anyway.

```

Comment:
This comment is a bit confusing. Perhaps: "This existing node was marked as invalid, so the caller must now re-render it."